userguide/snippets/usingui.rst

```diff
@@ -72,3 +72,15 @@ DNS hostname, or mDNS name:
   the %brand% system is at IP address 192.168.1.19, enter
   :samp:`ping {192.168.1.19}` on the command line of the other
   computer. If there is no response, check network configuration.
+
+
+.. _Help Text:
+
+Help Text
+~~~~~~~~~
+
+Every field and setting in the |web-ui| has a |help-text| icon.
+Additional information about the field or setting can be shown by
+clicking |help-text|. The help text is pinned to the screen until
```

Comment:
Not clear what "pinned" means here.  Maybe have the sentence about moving first:
The help text window can be dragged to any location, and will remain there until X or ? is clicked to close the window. Multiple help text windows can be open at one time.